tests/generation/test_generation_utils.py

```diff
@@ -1728,30 +1735,40 @@ def test_contrastive_search_bart(self):
         self.assertListEqual(
             generated_text,
             [
-                """Liana Barrientos, 39, pleaded not guilty to two counts of "offering a false instrument" Prosecutors say the marriages were part of an immigration scam. In total, Barriento has been married 10 times, with nine of her marriages occurring between 1999 and 2002."""
+                'Liana Barrientos, 39, pleaded not guilty to charges related to false marriage statements. '
```

Comment:
The input when this test was added had an extra space that was removed in the final commit. With the extra (incorrect) space, we get the original output.